src/CSET/operators/read.py

```diff
@@ -239,6 +242,19 @@ def _deterministic_callback(cube, field, filename):
         )
 
 
+def _um_normalise_callback(cube: iris.cube.Cube, field, filename):
+    """Normalise UM STASH variable long names to LFRic variable names.
+
+    Note UM standard names will remain associated with cubes where different.
+    Long_name will be used consistently in output filename and titles.
+    """
+    # Convert STASH to LFRic variable name
+    if "STASH" in cube.attributes:
+        stash = cube.attributes["STASH"]
+        (name, grid) = STASH_TO_LFRIC.get(str(stash), (None, "a"))
+        cube.long_name = name
```

Comment:
```suggestion
        try:
            (name, grid) = STASH_TO_LFRIC[str(stash)]
            cube.long_name = name
        except KeyError:
            logging.warning("Unknown STASH code: %s", stash)
            # Don't change cubes with unknown stash codes.
            pass
```

Currently if we don't have a variable in our dictionary we are removing the long name, which is undesirable. We should instead leave it unchanged. Logging might also be helpful, as it will allow us to expand our dictionary as needed.